### ISSUE TITLE
Centered bullets look strange for large paragraphs

### ISSUE THREAD

**Issue by drewvolz** [StoDevX/AAO-React-Native#2159]
Head to FAQs to see an example. The bullet is positioning itself in the middle of the paragraph rather than at the top.

I would like to kill the bullets on FAQs altogether but that’s beyond the scope of this issue.

**Comment by drewvolz** [comment]
And on other views like 2-line sentence bullets on privacy policy

**Comment by hawkrives** [comment]
Yes. But they look better imho than being stuck at the very top of the list item.

**Comment by drewvolz** [comment]
We think differently. I feel that in the middle looks out of place for a large list item.

**Comment by hawkrives** [comment]
I agree. 

I think you've forgotten what it used to look like.

**Comment by rye** [comment]
2-line bullets look fine to me.  Really, 2–4 lines is good.  Beyond that, we'd be missing the point of using bullets.

As long as there's a visual break between bullets, (i.e. after the bulleted paragraph, there is a short break not more than one line-height's worth) I don't think it's misleading to have the bullet on the side.  Screenshots of overtly offensive examples would be appreciated, but I don't have anything against the idea of putting the bullets aligned properly with the top line, as one might expect.

**Comment by hawkrives** [comment]
I would love to have the bullet alighted with baseline of the top line, as you'd expect it to be. 

However, the only way I could come up with to accomplish that feat was absolute positioning, which breaks when the user has a different font size.

**Comment by drewvolz** [comment]
Here’s how Apple did theirs
<img src="https://user-images.githubusercontent.com/5240843/35294632-48260df2-0034-11e8-9021-ab0aa063813b.png" width=340 />

**Comment by hawkrives** [comment]
Yes, well, I detest Apple's design there, so I am absolutely not going to follow it. :-1:

**Comment by drewvolz** [comment]
@rye and I feel differently. I think we need to have a continued discussion about this.

**Comment by hawkrives** [comment]
IIRC, you disliked the centered bullets. I also dislike the centered bullets, but I prefer them to first-line-indent-only bullets.

I believe that you would prefer

- Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

versus

• Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

--- 

edit: Am I incorrect?

**Comment by rye** [comment]
Backing @hawkrives up on that one. It completely defeats the purpose of a bullet point if you wrap back past the indentation on the bullet point.

**Comment by drewvolz** [comment]
Apple’s design is bad here, yeah. But our current impl is not an improvement.

**Comment by drewvolz** [comment]
And you’re correct that I prefer the first example

**Comment by drewvolz** [comment]
Closing due to the fact that our bullets work fine.

**Comment by hawkrives** [comment]
Reopening because they still look strange

**Comment by hawkrives** [comment]
resolved by moving to https://github.com/software-mansion/enriched-markdown
